### ISSUE TITLE
Other Windows over Minecraft Window

### ISSUE THREAD

**Issue by htmlmason** [comp500/BorderlessMining#82]
I want to have other windows, such as my iTunes MiniPlayer above the Minecraft Window. I have my other window pinned to the top using deskpins. How do I stop the other window from disappearing when Minecraft is focused?

**Comment by yoyotam3** [comment]
Yeah this feature would be nice, I downloaded the mod so I could use OnTopReplica to watch videos while playing but it seems like that doesn't work even though it works in other games that have native Borderless Windowed modes
